src/plots/cartesian/layout_attributes.js

```diff
@@ -498,6 +493,10 @@ module.exports = {
         ].join(' ')
     },
 
+    _nestedModules: {
```

Comment:
this tell `Plotly.PlotSchema.get` to look for the attributes in the component registry.